src/runtimes/wapc.rs

```diff
@@ -14,20 +21,20 @@ use crate::cluster_context::ClusterContext;
 use crate::policy::Policy;
 use crate::policy_evaluator::{PolicySettings, ValidateRequest};
 
-use kubewarden_policy_sdk::host_capabilities::{
-    crypto_v1::{CertificateVerificationRequest, CertificateVerificationResponse},
-    SigstoreVerificationInputV1, SigstoreVerificationInputV2,
-};
-use kubewarden_policy_sdk::metadata::ProtocolVersion;
-use kubewarden_policy_sdk::response::ValidationResponse as PolicyValidationResponse;
-use kubewarden_policy_sdk::settings::SettingsValidationResponse;
-use tokio::sync::oneshot::Receiver;
+pub(crate) struct Runtime<'a>(pub(crate) &'a mut WapcStack);
 
 lazy_static! {
     pub(crate) static ref WAPC_POLICY_MAPPING: RwLock<HashMap<u64, Policy>> =
         RwLock::new(HashMap::with_capacity(64));
 }
 
+/// Error message returned by wasmtime_provider when the guest execution
+/// is interrupted because of epoch deadline is exceeded.
+///
+/// Unfortunately, wasmtime_provider doesn't return a typed error, hence we have
+/// to look for this text
+const WAPC_EPOCH_INTERRUPTION_ERR_MSG: &str = "guest code interrupted, execution deadline exceeded";
```

Comment:
should we add a unit test that triggers this error, and checks that the string keeps being this one? I'm scared it subtly changes in future versions of wasmtime_provider and we just skip it.

Reply:
This is covered by the e2e tests added to policy-server, but yes... this is something we should cover with a quick test also there. I'm going to fix that